meshdb/meshdb.go

```diff
@@ -84,10 +100,16 @@ func NewMeshDB(path string) (*MeshDB, error) {
 		return nil, err
 	}
 
+	metadata, err := setupMetadata(database)
+	if err != nil {
+		return nil, err
+	}
+
 	return &MeshDB{
 		database:    database,
 		MiniHeaders: miniHeaders,
 		Orders:      orders,
+		Metadata:    metadata,
```

Comment:
Let's make the `Metadata` field unexported at this point, since users of the `meshdb` package should only need to two methods: `GetMetadata` and `SaveMetadata`.